### PR TITLE
Feature/sqlalchemy integration

### PR DESCRIPTION
**Pull request by default-usr**

### CODE REVIEW

File: thundra/plugins/trace/trace_plugin.py [hunk 1]
```diff
@@ -51,6 +52,7 @@ def before_invocation(self, plugin_context):
                                                    trace_id=trace_id,
                                                    transaction_id=transaction_id)
         self.root_span = self.scope.span
+        trace_support.root_span_started = True
```
Comment (serkan-ozal) [inline thundra/plugins/trace/trace_plugin.py]: Shouldn't this be reset to `False` after every invocation?

Comment (default-usr) [reply thundra/plugins/trace/trace_plugin.py]: Yes it should, will fix it

Comment (serkan-ozal) [reply thundra/plugins/trace/trace_plugin.py]: Maybe better approach might be using `get_active_span` over `ThundraTracer`. So if there is any active span, we can proceed instead introducing another variable `root_span_started`.

File: thundra/integrations/sqlalchemy.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+from thundra.opentracing.tracer import ThundraTracer
+
+from sqlalchemy.event import listen
+from sqlalchemy.engine.interfaces import ExecutionContext
+
+logger = logging.getLogger(__name__)
+
+
+class SqlAlchemyIntegration(RdbBaseIntegration):
+    CLASS_TYPE = 'sqlalchemy'
```
Comment (serkan-ozal) [inline thundra/integrations/sqlalchemy.py]: I think it is better to define as `SQLAlchemy` since as far as I know it is used directly on the UI

**Pull request merged**